pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8606.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="Dmitrii Eliseev" id="2022-08-17-mail-template">
+        <createTable tableName="mail_template">
+            <column name="mail_template_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="mail_template_pk"/>
+            </column>
+            <column name="content_type" type="VARCHAR(1000)">
+                <constraints nullable="false" />
+            </column>
+            <column name="subject" type="TEXT">
+                <constraints nullable="false" />
+            </column>
+            <column name="body" type="TEXT">
+                <constraints nullable="false" />
+            </column>
+            <column name="name" type="VARCHAR(500)">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+
+        <addUniqueConstraint tableName="mail_template" columnNames="name" constraintName="mail_template_unique" />
+
+        <createTable tableName="mail_template_repeatable_element">
+            <column name="mail_template_repeatable_element_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="mail_template_repeatable_element_pk"/>
+            </column>
+            <column name="mail_template_id" type="bigint">
+                <constraints references="mail_template(mail_template_id)" foreignKeyName="mail_template_repeatable_element_mail_template_fk"/>
+            </column>
+            <column name="name" type="VARCHAR(500)">
+                <constraints nullable="false" />
+            </column>
+            <column name="content" type="TEXT">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+
+        <addUniqueConstraint tableName="mail_template_repeatable_element"
+                             columnNames="mail_template_id, name"
+                             constraintName="mail_template_repeatable_element_unique" />
+    </changeSet>
+
+    <changeSet author="Dmitrii Eliseev" id="2022-08-17-mail-template-for-singular">
```

Comment:
Ideally this changes that inserts data should be in separate file under db-changes/seed/* rather than schema but we have this all over any way.

Reply:
I'd argue that this data doesn't even belong in the `seed` directory- that directory is intended for inserts which are for general use by the platform. This is _highly_ study specific, and the ideal spot for this is in the `singular` study configuration

Reply:
Yeah, as I mentioned in the ticket description, the right way is to move this to study configuration. But it is quite a big chunk of work that is not so important at the moment I believe.

Reply:
How about this- leave the `2022-08-17-mail-template` changeset where it is- we can deal with this later. Remove the `2022-08-17-mail-template-for-singular` changeset and add it as a patch to Singular. It's not a perfect solution, but it does keep study-specific changes in a more appropriate place.

Reply:
Sounds good Blake.